test/tests/services/has-many-dissociator.test.js

```diff
@@ -0,0 +1,124 @@
+import mongoose from 'mongoose';
+import loadFixture from 'mongoose-fixture-loader';
+import Interface from 'forest-express';
+import mongooseConnect from '../../utils/mongoose-connect';
+import HasManyDissociator from '../../../src/services/has-many-dissociator';
+
+describe('service > has-many-dissociator', () => {
```

Comment:
[Wording] `Service > hasManyDissociator` looks better. And I think we already formatted the other like that

Reply:
for all other services, the dashed version is used in the tests, so I used it to align with others

Reply:
Okey my bad